dao/FashionKnex.js

```diff
@@ -70,6 +71,9 @@ class FashionKnex {
   async insert (data, {returning = '*'} = {}) {
     data = _clean(data)
 
+    // If an id is not set upon insert, set one
+    if (!data.id) data.id = uuid.v4()
```

Comment:
This is related to my above comment in `Entity.js`. The id creation will happen on `insert` now. If an `Entity` without an `id` property is passed to `insert`, we will assign one automatically.